packages/config/config/devices/0x0086/zw090.json

```diff
@@ -33,90 +36,96 @@
 			"label": "Enable/disable the LED indicator",
-			"description": "This parameter is used to enable/disable the LED indicator.",
+			"description": "This parameter is used to enable/disable the LED indicator",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 1,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
-				{
-					"label": "Disable",
-					"value": 0
-				},
 				{
 					"label": "Enable",
 					"value": 1
+				},
+				{
+					"label": "Disable",
+					"value": 0
 				}
 			]
 		},
 		"220": {
-			"label": "RF power level",
-			"description": "RF power level",
+			"label": "Configure the RF power level",
+			"description": "There are 10 levels can be configured for the RF power level",
 			"valueSize": 1,
-			"minValue": 1,
+			"minValue": 0,
 			"maxValue": 10,
 			"defaultValue": 10,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": true
 		},
 		"242": {
-			"label": "Security network enabled",
-			"description": "Security network enabled",
+			"label": "Enable/disable Security network",
+			"description": "This parameter is used to enable/disable security network",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "disable",
+					"label": "Disable",
 					"value": 0
 				},
 				{
-					"label": "enable",
+					"label": "Enable",
 					"value": 1
 				}
 			]
 		},
-		// This parameter has an invalid valueSize
-		//"243": {
-		//	"label": "Security Network Key",
-		//	"description": "Security Network Key",
-		//	"unit": "key",
-		//	"valueSize": 16,
-		//	"minValue": 0,
-		//	"maxValue": 0,
-		//	"defaultValue": 0,
-		//	"readOnly": false,
-		//	"writeOnly": false,
-		//	"allowManualEntry": true
-		// },
 		"252": {
-			"label": "Lock Configuration",
-			"description": "Configuration lock. 1=enable, 0=disable",
+			"label": "Enable/disable the configuration parameters to be locked",
+			"description": "This parameter is used to Enable/disable the configuration parameters to be locked.\n(0== disable, 1== enable)",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
 			"defaultValue": 0,
+			"unsigned": true,
 			"readOnly": false,
 			"writeOnly": false,
 			"allowManualEntry": false,
 			"options": [
 				{
-					"label": "disable",
+					"label": "Disable",
 					"value": 0
 				},
 				{
-					"label": "enable",
+					"label": "Enable",
 					"value": 1
 				}
 			]
+		},
+		"255": {
+			"label": "Reset the configuration parameter to default factory setting",
+			"description": "Resets all configuration parameters to default setting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 0,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
```

Comment:
The Aeotec devices usually have some weird value (not 0) that needs to be put here.

Reply:
Some do, but not all. The zw089 also has only 0/1. From the manual.

![Screen Shot 2021-01-26 at 12 08 39 AM](https://user-images.githubusercontent.com/33612110/105803097-b7d08400-5f6a-11eb-9886-c38eedaab8e4.png)